api/src/Tests/Cases/VolunteersTest.php

```diff
@@ -126,13 +126,26 @@ public function testHours(): void
         $this->checkHourEntry($data, 2);
 
         /* Overlap Testing */
-        $base = date('Y-m-d h:i:s', strtotime('+1 Year'));
-        $plusMin = date('Y-m-d h:i:s', strtotime('+1 Year +30 minutes'));
-        $minusMin = date('Y-m-d h:i:s', strtotime('+1 Year -30 minutes'));
-        $plusHour = date('Y-m-d h:i:s', strtotime('+1 Year +1 hour'));
-        $minusHour = date('Y-m-d h:i:s', strtotime('+1 Year -2 hours'));
-        $largeEnd = date('Y-m-d h:i:s', strtotime('+1 Year +4 hours'));
-        $subEnd = date('Y-m-d h:i:s', strtotime('+1 Year -30 minutes'));
+        $date = new \DateTime('+1 Year');
```

Comment:
Since the date intervals are specified as strings, could this be an array of date intervals and a loop?

Reply:
I know. It is strangely picky.